### PR TITLE
Update Linea

### PR DESCRIPTION
**Pull request by sdlyy**
Resolves L2B-3469

## Changes

New role has been configured (`1`).
One and only member has been assigned ([0x453B3A4b4d64B4E6f472A306c3D4Fc318C34bbA8](https://etherscan.io/address/0x453B3A4b4d64B4E6f472A306c3D4Fc318C34bbA8)).

Assigned member can now invoke:

- `pauseByType(bytes32)` on ZkEVM/MessageService ([0xd19d4B5d358258f05D7B411E21A1460D11B0876F](https://etherscan.io/address/0xd19d4B5d358258f05D7B411E21A1460D11B0876F))
- `pause()` on ERC20Bridge ([0x051F1D88f0aF5763fB888eC4378b4D8B29ea3319](https://etherscan.io/address/0x051F1D88f0aF5763fB888eC4378b4D8B29ea3319))
- `pause()` on USDCBridge ([0x504A330327A089d8364C4ab3811Ee26976d388ce](https://etherscan.io/address/0x504A330327A089d8364C4ab3811Ee26976d388ce))

## Notes

- [Role configuration transaction](https://etherscan.io/tx/0x713fbc3ad326969596c6d7fb38e76081a971036333b3807f2302242d87b53134)
- Tenderly stack
image

I'm not into the topic much yet, so I have to sync my understanding of zodiac Roles w/ somebody.
I presume this one member cannot pause the bridge without owners signing the transactions first ;p

I'll write the extractor for roles to be displayed on the page once I get confirmation.

### CODE REVIEW

**Comment by mateuszradomski** [comment]
Nope, just that address has the same power over those three functions as the main multi-sig. There is no need for any signatures from the multi-sig, the `Roles` module bypasses it...

File: packages/config/src/layer2s/linea.ts [hunk 1]
```diff
@@ -298,23 +296,28 @@ export const linea: Layer2 = {
...
+    {
+      accounts: [
+        {
+          address: EthereumAddress(
+            '0x453B3A4b4d64B4E6f472A306c3D4Fc318C34bbA8',
+          ),
+          type: 'EOA',
+        },
+      ],
+      name: 'Members of Role 1',
```
Comment (lucadonnoh) [inline packages/config/src/layer2s/linea.ts:317]:
```suggestion
      name: 'Pauser',
```

File: packages/config/src/layer2s/linea.ts [hunk 2]
```diff
@@ -298,23 +296,28 @@ export const linea: Layer2 = {
...
       name: 'Operators',
       description:
         'The operators are allowed to prove blocks and post the corresponding transaction data.',
     },
+    // FIXME: Add value transformer
+    {
+      accounts: [
+        {
+          address: EthereumAddress(
+            '0x453B3A4b4d64B4E6f472A306c3D4Fc318C34bbA8',
```
Comment (lucadonnoh) [inline packages/config/src/layer2s/linea.ts:312]: don't hardcode the address

File: packages/config/src/layer2s/linea.ts [hunk 3]
```diff
@@ -298,23 +296,28 @@ export const linea: Layer2 = {
...
+          ),
+          type: 'EOA',
+        },
+      ],
+      name: 'Members of Role 1',
+      description:
+        'Allowed to invoke pause() on ERC20Bridge and USDCBridge. Allowed to invoke pauseByType() on zkEVM contract.',
```
Comment (lucadonnoh) [inline packages/config/src/layer2s/linea.ts:319]:
```suggestion
        'Address allowed to pause the ERC20Bridge, the USDCBridge and the core functionalities of the project.',
```

**Review by lucadonnoh — CHANGES_REQUESTED** [review]

File: packages/config/src/layer2s/linea.ts (hunk 1 above)
Comment (sdlyy) [reply packages/config/src/layer2s/linea.ts]: [chore: remove hardcoded zodiac pauser](https://github.com/l2beat/l2beat/pull/2565/commits/565a7714a4939e24e98f52e493a20906fd29632d)

File: packages/config/src/layer2s/linea.ts (hunk 2 above)
Comment (sdlyy) [reply packages/config/src/layer2s/linea.ts]: [chore: remove hardcoded zodiac pauser](https://github.com/l2beat/l2beat/pull/2565/commits/565a7714a4939e24e98f52e493a20906fd29632d)

File: packages/config/src/layer2s/linea.ts [hunk 4]
```diff
@@ -50,6 +48,17 @@ const roles = discovery.getContractValue<{
...
+const zodiacRoles = discovery.getContractValue<{
+  roles: Record<string, Record<string, boolean>>
+}>('Roles', 'roles')
+const zodiacPauserRole = '1'
+const zodiacPausers: ScalingProjectPermissionedAccount[] = Object.keys(
+  zodiacRoles.roles[zodiacPauserRole].members,
+).map((zodiacPauser) => ({
+  address: EthereumAddress(zodiacPauser),
+  type: 'EOA',
+}))
```
Comment (michalsidzej) [inline packages/config/src/layer2s/linea.ts:60]: Please use `discovery.formatPermissionedAccount` or assert that those pausers are indeed eoas here.

**Review by michalsidzej — CHANGES_REQUESTED** [review]

File: packages/config/src/layer2s/linea.ts (hunk 4 above)
Comment (sdlyy) [reply packages/config/src/layer2s/linea.ts]: [chore: use discovery formatter for zodiac pausers](https://github.com/l2beat/l2beat/pull/2565/commits/4cb77c1eccabd04629aa6aadf70d7b0127bbca02)

**Review by lucadonnoh — APPROVED** [review]

**Pull request merged**